dracut.sh

```diff
@@ -2327,6 +2338,7 @@ fi
 if ! [[ $compress ]]; then
     # check all known compressors, if none specified
     for i in $DRACUT_COMPRESS_PIGZ $DRACUT_COMPRESS_GZIP $DRACUT_COMPRESS_LZ4 $DRACUT_COMPRESS_LZOP $DRACUT_COMPRESS_ZSTD $DRACUT_COMPRESS_LZMA $DRACUT_COMPRESS_XZ $DRACUT_COMPRESS_LBZIP2 $DRACUT_COMPRESS_BZIP2 $DRACUT_COMPRESS_CAT; do
+        [[ $i != "$DRACUT_COMPRESS_ZSTD" || $DRACUT_KERNEL_RD_ZSTD ]] || continue
```

Comment:
or just unset DRACUT_COMPRESS_ZSTD if `check_kernel_config CONFIG_RD_ZSTD` fails

Reply:
> or just unset DRACUT_COMPRESS_ZSTD if `check_kernel_config CONFIG_RD_ZSTD` fails

I can do that if you want, but tbh it feels a little dirty to me.

Reply:
true